Tests/ASDisplayNodeTests.mm

```diff
@@ -1922,18 +1923,23 @@ - (void)checkBackgroundColorOpaqueRelationshipWithViewLoaded:(BOOL)loaded layerB
   }
 
   XCTAssertTrue(node.opaque, @"Node should start opaque");
-  XCTAssertTrue(node.layer.opaque, @"Node should start opaque");
+  if (isLayerBacked) {
+      XCTAssertTrue(node.layer.opaque, @"Set background color should not have made this layer not opaque");
+    } else {
+      XCTAssertTrue(node.view.opaque, @"Set background color should not have made this view not opaque");
+    }
 
   node.backgroundColor = [UIColor clearColor];
+//  [node.view setNeedsDisplay];
```

Comment:
Wondering the reason behind having this line in the first place. Would that be because doing this causes the view to sync its states (opaque and background color) to the layer? Even if that's true, I think we still shouldn't rely on it because of what I said in the preview comment.

Plus, we should just remove this line.

Reply:
i meant to remove this!